src/main/java/seedu/address/model/person/FieldContainsKeywordsPredicate.java

```diff
@@ -58,7 +55,7 @@ private boolean testName(Person person) {
 
     private boolean testJob(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getJobTitle().toString(), keyword));
+                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getJobTitle().get().jobTitle, keyword));
```

Comment:
Could this cause a `NullPointerException` when the `jobTitle` is `null`?

Reply:
Same for `company`

Reply:
Thanks for pointing out, will resolve